src/Editor/GUI/Editors/EntityEditor.hpp

```diff
@@ -6,6 +6,7 @@
 #include <functional>
 #include <Engine/Entity/Entity.hpp>
 #include <imgui.h>
+#include <Engine/Geometry/AssetFileHandler.hpp> 
```

Comment:
There is no other change in this file (the header) so we should prefer to place this include in the cpp file instead.